packages/ckeditor5-engine/src/model/documentselection.js

```diff
@@ -364,6 +366,17 @@ export default class DocumentSelection {
 		this._selection._updateAttributes( false );
 	}
 
+	/**
+	 * Registers marker group prefix to be collected in {@link ~DocumentSelection#markers selection markers collection}.
+	 *
+	 * See also {@link module:engine/model/markercollection~MarkerCollection#getMarkersGroup `MarkerCollection#getMarkersGroup()`}.
+	 *
+	 * @param {String} prefix Marker group prefix.
+	 */
+	observeMarkersGroup( prefix ) {
```

Comment:
How about `groupName` instead. of prefix, if we use "markers group" everywhere?. Also could you add an example in the API docs? I know this is straightforward but "marker group" is a loose term that is introduced only in the API docs, this is not a property of a marker or something, so someone may be lost what it means.

Reply:
That's why I linked to `MarkersCollection` - there is `prefix` so I kept the same convention (and this seems more reasonable than `groupName` because it is not confusing with any `Marker` property.

Reply:
You are right - we use prefix there.